src/Adapter/Module/AdminModuleDataProvider.php

```diff
@@ -57,6 +57,21 @@ class AdminModuleDataProvider implements ModuleInterface
      */
     protected $moduleActions = array('install', 'uninstall', 'enable', 'disable', 'enable_mobile', 'disable_mobile', 'reset', 'upgrade');
 
+    /**
+     * @var array giving a translation domain key for each module action
+     */
+    protected $actionsTranslationDomains = array(
+        'install' => 'Admin.Actions',
+        'uninstall' => 'Admin.Actions',
+        'enable' => 'Admin.Actions',
+        'disable' => 'Admin.Actions',
+        'enable_mobile' => 'Admin.Modules.Feature',
+        'disable_mobile' => 'Admin.Modules.Feature',
+        'reset' => 'Admin.Actions',
+        'upgrade' => 'Admin.Actions',
+        'configure' => 'Admin.Actions',
+    );
+
```

Comment:
I'm not so sure I'm doing this at the right place, if anyone as a better suggestion I'll be willing to discuss and implement it !

Reply:
If you don't change its value, use const instead

Reply:
done !